### DISCUSSION TITLE
what is the goal of the emscripten generated file: myProject.Emscripten.worker.js ?

### DISCUSSION THREAD

**Discussion by aharondavid** [emscripten-core/emscripten#17455] [Q&A]
Hi,
what is the goal of the emscripten generated file: `myProject.Emscripten.worker.js`  ?
should include it in my main web page ?

**Comment by sbc100** [comment]
This file is the one that emscripten uses to start new pthreads/workers.  If you build with `-sUSE_PTHREADS` you will see that file but otherwise you should not have it.   It needs to be available alongside the regular `myProject.Emscripten.js` files but the loading of this file is taken care of for you, so you don't need to include it on your main page.

**Comment by aharondavid** [comment]
Ok, Thank you!
